safor/index.html

```diff
@@ -33,23 +33,23 @@ <h1>switching your <br id="switching-your-br">life to safety mode</h1>
         <div class="video">
             <h1>WHAT IS SAFOR?</h1>
             <div id="playbutton">
-                <img src="./images/section2/playbutton.svg" alt="playbutton">
+                <img src="images/section2/playbutton.svg" alt="playbutton">
                 <p>WATCH NOW</p>
             </div>
         </div>
-        <div class="text">
-            <p>The existing types of protection in today's society can not always be adapted to the needs of the modern customers.</p>
-            <p>The SAFOR system is flexible and adaptive, providing personal protection to you and your loved once  at any time of the day, wherever they are.</p>
-            <p>Regardless the situation, the system has unlimited possibilities to make your life free and safe.</p>
-        </div>
+        <ul class="text">
```

Comment:
Something went wrong here. Please fix positioning. Also nesting p tag into li makes no sense. You can just put text in li item and style it as p.